proto/buf/alpha/registry/v1alpha1/role.proto

```diff
@@ -57,3 +57,15 @@ enum PluginRole {
   PLUGIN_ROLE_WRITE = 3;
   PLUGIN_ROLE_READ = 4;
 }
+
+message RepositoryRoleValue {
+  RepositoryRole value = 1;
+}
+
+message PluginRoleValue {
+  PluginRole value = 1;
+}
+
+message TemplateRoleValue {
+  TemplateRole value = 1;
+}
```

Comment:
@mfridman reminded me that as our enums always have an unspecified 0 value without semantic meaning, we don't need the wrapper values here.

I think we don't actually need these here, my bad for putting you on the wrong track.

Reply:
I liked your suggestion though, and it does keep it consistent with `google.protobuf.StringValue` and co.

I think we should keep it as is.. because it does give the flexibility to differentiate between unset vs unspecified, despite today us not having semantic meaning for the "zero" value.

Reply:
yes, I like the idea of differentiate between unset vs unspecified, cause in our codebase the unspecified 0 value is usually meaning that it is invalid, while an nil *Value is a valid unset value

Reply:
https://docs.buf.build/lint/rules#enum_zero_value_suffix in our own docs does suggest that this 0 value should always be something without specified meaning, specifically so you can detect if it was explicitly set or not. 

Unlike some of the others covered by the well known types (bool, string, ints), enums have a specific value which we treat as "unspecified"/unset. If we wouldn't have this value I would agree that the wrapper type makes sense, but given that we already have a special 0 value I think we probably shouldn't also have a wrapper.

Reply:
Ye that's a fair point. The enum value shouldn't convey meaning and its best practice to _always_ have an unspecified value as 0.

In the event we wanted to convey something like "none" then we'd need to introduce a new (valid) enum, but this seems like a reasonable tradeoff.

We should also add this to our company style guide